test/common/testlib.py

```diff
@@ -2099,6 +2099,8 @@ def enable_root_login(self):
         disallowed_conf = '/etc/cockpit/disallowed-users'
         if not self.machine.ostree_image and self.file_exists(disallowed_conf):
             self.sed_file('/root/d', disallowed_conf)
+
```

Comment:
` Add white lines ` Why? :) Is this some linter that you use?

Reply:
Nothing like that, but @jelly instructed me not to drop the white lines there.